docs/src/main/asciidoc/stork.adoc

```diff
@@ -199,8 +199,8 @@ public class Registration {
     @ConfigProperty(name = "consul.host") String host;
     @ConfigProperty(name = "consul.port") int port;
 
-    @ConfigProperty(name = "blue-service-port", defaultValue = "9000") int red;
-    @ConfigProperty(name = "red-service-port", defaultValue = "9001") int blue;
+    @ConfigProperty(name = "red-service-port", defaultValue = "9000") int red;
+    @ConfigProperty(name = "blue-service-port", defaultValue = "9001") int blue;
```

Comment:
Minor comment but given we are changing the order here, could we also change the order in which the services are registered a few lines below? That would be less confusing IMHO.